src/plots/errorBarPlot.ts

```diff
@@ -0,0 +1,115 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Plots {
+  export class ErrorBar<X, Y> extends XYPlot<X, Y> {
```

Comment:
I think we went through an RFC the first time around re: the naming, the conclusion was that since these things are so commonly called Error Bars, we should just stick to this naming.